HW_2/task_2.py

```diff
@@ -0,0 +1,32 @@
+"""
+Задание 2.
+Напишите программу, которая получает целое число и возвращает его шестнадцатеричное строковое представление.
+Функцию hex используйте для проверки своего результата.
+"""
+
+
+def convert_to_hex(int_num):
+    BASE = 16
+    dictionary = {0: '0', 1: '1', 2: '2', 3: '3', 4: '4', 5: '5',
+                  6: '6', 7: '7', 8: '8', 9: '9', 10: 'a',
+                  11: 'b', 12: 'c', 13: 'd', 14: 'e', 15: 'f'}
```

Comment:
Интересно то, что если просто заменить это словарь на строку, но остальной код будет работать также без изменений
```suggestion
    dictionary = '0123456789abcdef'
```